op.c

```diff
@@ -10814,7 +10814,9 @@ S_process_special_blocks(pTHX_ I32 floor, const char *const fullname,
                 sv_setiv(max_nest_sv, max_nest_iv);
             }
 
-            if (PL_eval_begin_nest_depth >= max_nest_iv) {
+            /* (UV) below is just to silence a compiler warning, and should be
+             * effectively no-op as max_nest_iv will never be negative here */
```

Comment:
Change to
effectively a no-op

Reply:
s/effectively no-op/effectively a no-op,/

Reply:
Thank you for your advise.  I've pushed a commit to fix it.